### PR TITLE
[5.7] Add missing dependency

### PR DESCRIPTION
**Pull request by phannaly**
Add missing dependency inside http.

### CODE REVIEW

**Comment by browner12** [comment]
can you point out some code examples of where these are required?

**Comment by phannaly** [comment]
@browner12  in `Response.php` use `Illuminate\Contracts`​ and in `UploadedFile.php` use `Illuminate\Container`.

**Comment by GrahamCampbell** [comment]
Could you add the container dependency only the suggest list, instead of require please.

File: src/Illuminate/Http/composer.json [hunk 1]
```diff
@@ -32,6 +31,9 @@
             "dev-master": "5.7-dev"
         }
     },
+    "suggest": {
+        "illuminate/container": "Required to use the container (5.7.*)."
```
Comment (GrahamCampbell) [inline src/Illuminate/Http/composer.json]: Could you specify what exactly needs the container, like the other suggest things (take a look at the other composer.json files for some inspiration :))

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Http/composer.json (hunk 1 above)
Comment (phannaly) [reply src/Illuminate/Http/composer.json]: Yeah, I try to find suggestion `lluminate/container` from other but can't find, other lluminate only require `container` no `suggest`

Comment (GrahamCampbell) [reply src/Illuminate/Http/composer.json]: You'll need to look at how the package is used in this particular case to come up with a description. I only suggested looking at the other composer.json files to give you an example of other messages, so you know how to style it.

**Pull request closed without merging**